version/description

```diff
@@ -1,4 +1,2 @@
-0.53.4
+Release 0.53.5
```

Comment:
maybe i had to leave all the commits in the commit amend message ?

i see it removed them also from here

```
'* bump ovs-cni to v0.19.2 (#918)
* ovs-cni, release-0.53, Set source as tagged (#917)
* bump nmstate to v0.47.2 (#914)
* bump bridge-marker to 0.6.1 (#916)
* bridge-marker, release-0.53, Set source as tagged (#915)
* macvtap, release-0.53, Bump to version 0.5.0 (#913)
* release-0.53, Update k8s-nmstate to use tagged policy (#912)
* bump kubemacpool to v0.26.5 (#911)
* linux-bridge, Add Priority class (#908)
* operator, Add Priority class (#907)
* multus, Add Priority class (#909)
* components, Update multus repo url (#903)'
```

Reply:
are you sure you initiated the `make prepare-patch` on the most recent `release-0.53` branch commit?

Reply:
yes, it did show the list above, i simply removed them when it gives you the edit feature
we can see i have 918 which is the newest (this PR is 919)
```
commit c8a109fdffc4aeaf8d1343b13b895608477f1286
Author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Date:   Mon Jun 28 13:57:03 2021 +0200

    bump ovs-cni to v0.19.2 (#918)
```

Reply:
The section here should include all the commits that were inserted in this release. 
please re-enter them.

Reply:
Done (by rerunning the make target)